filebeat/processors/annotate/kubernetes/indexing.go

```diff
@@ -0,0 +1,91 @@
+package kubernetes
```

Comment:
We have a little bit a mess with package names singular/plural. We started all new packages to be singular (like module) when it contains multiple packages with each package one module. `processors` initially contained all processors in one package I think, that is why it was called `processors` and not `processor`. It should be also changed in libbeat later but in filebeat we should directly start with `processor`.